src/constants/index.ts

```diff
@@ -27,3 +27,7 @@ export const itemsLoadLimit = {
   mobile: 6,
   default: 12
 }
+
+export const sliceNames = {
+  cooperations: 'cooperationsSlice'
```

Comment:
Maybe this object should be moved to `/redux` separate file like `constants.ts`, since it's only will be used in redux slices? What do you think?

Reply:
Created `redux.constants.ts` and moved object there. Thanks for suggestion